### ISSUE TITLE
Stream requests are canceled when trying to seeking backward livestream video

### ISSUE THREAD

**Issue by IWTBCAdeveloper** [videojs/video.js#7983]
### Description

Hi there! as the title, I don't know what problem it is so please help 

<img width="1431" alt="Screenshot 2022-11-01 at 16 41 02" src="https://user-images.githubusercontent.com/13016058/199207035-c618ae16-a137-48b7-958a-0409bc917c2a.png">

### Steps to reproduce

1. Live stream video
2. Seeking backward
3. Stream requests are canceled

### What version of Video.js are you using?

7.14.3

### What browser(s) including version(s) does this occur with?

106.0.5249.119 (Official Build)

### What OS(es) and version(s) does this occur with?

windows 11

**Comment by gkatsev** [comment]
Cancelled segments requests shouldn't be an issue necessarily, especially if they are happening around seeking. As you're seeking, the playback engine is trying to requests segments from the place you're seeking but if you move on, we don't want to continue downloading the segment so we cancel the request. Once you stop seeking, you should see segments that have loaded successfully.

For example, I loaded https://videojs-http-streaming.netlify.app/ and seeking around a bit after clicking play and saw this once I stopped seeking
![image](https://user-images.githubusercontent.com/535884/199359118-ac5a69c1-3cc7-4fbd-a979-26ae74cfb029.png)

Hope that helps.

**Comment by IWTBCAdeveloper** [comment]
yes, I know the behavior that you try to explain, but the problem is when the canceled occur I can't seek backward or forward the live stream video anymore, it seems like a bug? Please see the video bellow

https://user-images.githubusercontent.com/13016058/199370459-de993edd-8921-4bb0-8458-40cd3e608b42.mov

**Comment by IWTBCAdeveloper** [comment]
please help !
